src/generators.rs

```diff
@@ -2466,7 +2466,22 @@ pub fn complete_graph(
     weights: Option<Vec<PyObject>>,
     multigraph: bool,
 ) -> PyResult<graph::PyGraph> {
-    mesh_graph(py, num_nodes, weights, multigraph)
```

Comment:
We should probably update `mesh_graph` to call `complete_graph` to avoid code duplication

Reply:
Sure. Done in https://github.com/Qiskit/rustworkx/pull/772/commits/0717ca68f6f99f2aa12a6328106ba9c142db3d31.